public/search.js

```diff
@@ -0,0 +1,204 @@
+  var bullet = $("#bulletList");
+
+  //function when search button is clicked
+  $("#search").click(function(){
+    firstDoc = 0;
+    lastDoc = 10;
+    $("#next").css({"visibility": "visible"});
+    $("#prev").css({"visibility": "visible"});
+    displayDocs(firstDoc, lastDoc);
+  });
+
+  //function when the next button is clicked
+  $("#next").click(function()
+    {
+      firstDoc+=10;
+      lastDoc+=10;
+      displayDocs(firstDoc, lastDoc);
+    }
+  );
+
+  //function when the prev button is clicked
+  $("#prev").click(function()
+    {
+        firstDoc-=10;
+        lastDoc-=10;
+        displayDocs(firstDoc, lastDoc);
+    }
+  );
+
+  /**
+  * Display docs is a function that displays the physical and logcial documents
+  * corresponding to the collection selected in the dropdown box.
+  * For each document the System and Valid times are displayed
+  *
+  * Note: We create an array of all physical and logical documents, in order
+  *       to be able to display a range of documents (so we have only 10 documents
+  *       appearing per page)
+  *
+  * @param start: the index of the first document you want to display
+  * @param end: the index of the last document you want to display
+  */
+  function displayDocs( start, end)
+  {
+    $("#bulletList").empty();
+    var e = document.getElementById("dropdown");
+    var selectedColl = e.options[e.selectedIndex].value;
+
+    //call to get all documents (excluding .lsqt) from the collection selected in the drop down list
+    var docs = $.ajax(
+    {
+      url: "http://localhost:3000/v1/search?structuredQuery={%20%22search%22:{%20%22query%22:{%20%22and-not-query%22:%20{%20%22positive-query%22:%20{%20%22collection-query%22:%20{%20%22uri%22:%20[%20%22"+selectedColl+"%22%20]%20}%20},%20%22negative-query%22:%20{%20%22collection-query%22:%20{%20%22uri%22:%20[%20%22lsqt%22%20]%20}%20}%20}%20},%20%22options%22:{%20%22search-option%22:[%22unfiltered%22]%20}%20}%20}&format=json&pageLength=1000",
+        success: function(data, textStatus) {
+        console.log('got collections: ' + data);
+      },
+      error: function(jqXHR, textStatus, errorThrown) {
+        console.log('problem');
+      },
+      async: false,
+    });
+    docs = JSON.parse( docs.responseText );
+
+    //Checks and sets boundary points.
+    //Looks at the index of the first and last document (passed into the function)
+    //and disables or enables the next/previous buttons based on those indexes.
+    if( start <= 1 )
+    {
+      document.getElementById("prev").disabled = true;
+    }
+    else
+    {
+      document.getElementById("prev").disabled = false;
+    }
+
+    if( end >= docs.total)
+    {
+      document.getElementById("next").disabled = true;
+      end = docs.total;
+    }
+    else
+    {
+      document.getElementById("next").disabled = false;
+    }
+
+    var docLen = docs.total;
+    document.getElementById("numDocs").innerHTML = start.toString() + " to " + end.toString() + " of " + docLen;
```

Comment:
This currently displays "0 to 10 of 18" for a search I just did. 0-10 is 11 elements, although only 10 are shown. Please change so that we see "1 to 10 of 18", then "11 to 18 of 18" on the second page.